deploy-manage/users-roles/cluster-or-deployment-auth/managed-credentials-eck.md

```diff
@@ -66,13 +66,17 @@ If you are using the `elastic` user credentials in your own applications, they w
 To regenerate all auto-generated credentials in a namespace, run the following command:
 
 ```sh
-kubectl delete secret -l eck.k8s.elastic.co/credentials=true
+kubectl delete secret -l eck.k8s.elastic.co/credentials=true,common.k8s.elastic.co/type!=kibana
```

Comment:
This will no longer be true since ECK 3.2. We are automatically excluding the Kibana form the credentials label.